src/pages/workspace/accounting/netsuite/import/NetSuiteImportCustomFieldNew/NetSuiteCustomListPicker.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import useLocalize from '@hooks/useLocalize';
+import Navigation from '@libs/Navigation/Navigation';
+import type {CustomListSelectorType} from '@pages/workspace/accounting/netsuite/types';
+import CONST from '@src/CONST';
+import type {Policy} from '@src/types/onyx';
+import NetSuiteCustomListSelectorModal from './NetSuiteCustomListSelectorModal';
+
+type NetSuiteCustomListPickerProps = {
+    /** Current value of the selected item */
+    value?: string;
+
+    /** Current connected policy */
+    policy?: Policy;
+
+    /** Callback when the list item is selected */
+    onInputChange?: (value: string, key?: string) => void;
+
+    /** Id of the internalID input to be updated on input change */
+    internalIDInputID?: string;
+
+    /** Form Error description */
+    errorText?: string;
+};
+
+function NetSuiteCustomListPicker({value, policy, internalIDInputID, errorText, onInputChange = () => {}}: NetSuiteCustomListPickerProps) {
+    const {translate} = useLocalize();
+    const [isPickerVisible, setIsPickerVisible] = useState(false);
+
+    const showPickerModal = () => {
+        setIsPickerVisible(true);
+    };
```

Comment:
No need for variable if we're calling it in only one place below.

Reply:
For consistency with hidePickerModal. Removed it.